faq/doc-copy.php

```diff
@@ -4,15 +4,12 @@
 include_once($relPath.'pg.inc');
 include_once($relPath.'theme.inc');
 
-output_header('Proofreading Guidelines -- Copyright Renewal Books', NO_STATSBAR);
-?>
+$theme_args["css_data"] = "tt { color: red } pre { color: red }";
```

Comment:
`<tt>` isn't used anywhere in this file (anymore) so let's delete the style for it. I failed to clean that up in e06a364b1.